src/index.html

```diff
@@ -5,8 +5,363 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <div>
+        <a href="#">
+          <img
+            src="\images\logo.png"
+            alt="logo"
+            class="header__logo"
+          >
+        </a>
+      </div>
+      <nav class="header__navigation">
+        <ul class="header__list">
+          <li class="header__item">
+            <a href="#" class="header__link is-active">apple</a>
+          </li>
```

Comment:
Create separate block for navigation. Better name it `nav` and `list, item, link` would be an elements of `nav` block.
Create separate scss file for `nav` block.
And add changes to styles according to changes in html.
```suggestion
      <nav class="nav header__navigation">
        <ul class="nav__list">
          <li class="nav__item">
            <a href="#" class="nav__link is-active">apple</a>
          </li>
```